users/views.py

```diff
@@ -101,15 +99,15 @@ def get_success_url(self) -> str:
 class UserUpdate(UpdateView):
     template_name = "user_update.html"
     form_class = SimpleUserChangeForm
-    admins_form_class = AdminUserChangeForm
```

Comment:
Why do you delete it and move it directly to method?

Reply:
The only explanation I can think of right now is totally ludicrous...
... but it might be that I was so fixated on this thing not being implemented, that I totally just ignored the existing (and working!) implementation and wrote the exact same thing from scratch...
And it turns out that this was only a template issue... I feel like I should ask for a reverse-raise or something. 😖

Reply:
I've changed it back to the previous state.